streams/src/test/java/org/apache/kafka/test/StreamsTestUtils.java

```diff
@@ -57,6 +57,13 @@ public static Properties getStreamsConfig(String bootstrapServer, String keySerd
                 new Properties());
     }
 
+    public static Properties minimalStreamsConfig() {
+        final Properties properties = new Properties();
+        properties.put(StreamsConfig.APPLICATION_ID_CONFIG, UUID.randomUUID().toString());
+        properties.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "blah:9092");
```

Comment:
Nit: can we change `blah` to `anyServer` or something less blah-y? Otherwise LGTM